warehouse/transformer/benchmark/main.go

```diff
@@ -58,16 +59,16 @@ func run(ctx context.Context) error {
 	logFactory := logger.Default
 	l := logFactory.NewLogger().Child("warehouse-transformer-benchmark").Child(mode)
 
-	clientEvents := make([]types.TransformerEvent, eventsInBatch)
-	for i := 0; i < len(clientEvents); i++ {
-		var transformerEvent types.TransformerEvent
-		err := jsonrs.Unmarshal([]byte(sampleEvent), &transformerEvent)
-		if err != nil {
-			return fmt.Errorf("could not unmarshal sample client event: %w", err)
-		}
-		clientEvents[i] = transformerEvent
+	var te types.TransformerEvent
```

Comment:
To confirm, the change in this file is specific to `samber/lo` and nothing to do with concurrency, rt ?